pkgs/development/python-modules/functorch/default.nix

```diff
@@ -29,6 +30,10 @@ buildPythonPackage rec {
     which
   ];
 
+  buildInputs = [
+    pybind11
+  ];
```

Comment:
I'm assuming this proved necessary in order to get the build to go through? I guess it was relying on PyTorch's pybind11 headers previously?

Reply:
correct, it was able to find pybind headers through pytorch

This seems to be the only package which had this behavior